src/components/TxNotification.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { EtherscanLink } from 'components/EtherscanLink'
```

Comment:
luckily with VS studio you can do `find and replace` and change all these at once

Reply:
You are using VS Studio? Why?

Reply:
What Is VS Studio, I only know vim.

jokes aside, VS studio hasn't being that good helping to rename imports when I "rename" or move a file around. Sometimes was "forgetting" some files to update, or he didn't do a `git mv` so git was detecting a "delete and a add" of a file

Reply:
Have you tried `Rename Symbol F2`? It renames across files

Reply:
Yep, that's how I normally do it. Also, moving the file renames the imports (when it works)